.github/workflows/nodejs-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Intacct JS SDK CI
```

Comment:
Nit: Probably just call it `Intacct Node.js SDK CI` to add more clarity about the Node.js we are using.